.sage/proto.go

```diff
@@ -11,6 +11,11 @@ import (
 
 type Proto sg.Namespace
 
+const (
+	// Version of buf.build/googleapis/googleapis to generate from.
```

Comment:
Maybe add the version is found under history https://buf.build/googleapis/googleapis/history

Reply:
will do!